### PR TITLE
Include a Changelog entry for agent priority

### PR DESCRIPTION
**Pull request by rafbgarcia**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,7 @@
 # [develop](https://github.com/adhearsion/electric_slide)
 
+* Bugfix: `FixedPriority` `#checkout_agent` now returns the agent with highest priority
```
Comment (rafbgarcia) [inline CHANGELOG.md]: Is this good @benlangfeld ?

Comment (benlangfeld) [inline CHANGELOG.md]: I would try to make this more friendly, like "Bugfix: FixedPriority now correctly checks out agents in priority order."

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,7 @@
 # [develop](https://github.com/adhearsion/electric_slide)
 
+* Bugfix: FixedPriority now correctly checks out agents in priority order.
```
Comment (rafbgarcia) [inline CHANGELOG.md]: done

File: lib/electric_slide/agent_strategy/fixed_priority.rb [hunk 3]
```diff
@@ -30,8 +30,7 @@ def available_agent_summary
       end
 
       def checkout_agent
-        priorities = @priorities.sort.to_h
-        _, agents = priorities.detect do |priority, agents|
+        _, agents = @priorities.sort.detect do |priority, agents|
```
Comment (rafbgarcia) [inline lib/electric_slide/agent_strategy/fixed_priority.rb]: Ops, I just noticed that you've made a comment on the other PR. Should I revert this?

Comment (benlangfeld) [inline lib/electric_slide/agent_strategy/fixed_priority.rb]: Nope, this is what I wanted it to look like :)

**Comment by benlangfeld** [comment]
:+1:

**Pull request merged**